Makefile

```diff
@@ -451,37 +450,30 @@ clean:
 	$(V0) rm -rf $(OBJECT_DIR)/$(TARGET)
 	@echo "Cleaning $(TARGET) succeeded."
 
-## clean_test        : clean up temporary / machine-generated files (tests)
-clean_test-%:
-	$(MAKE) clean_test
+## test_clean        : clean up temporary / machine-generated files (tests)
+test-%_clean:
+	$(MAKE) test_clean
 
-clean_test:
+test_clean:
 	$(V0) cd src/test && $(MAKE) clean || true
 
-## clean_<TARGET>    : clean up one specific target
-$(CLEAN_TARGETS):
-	$(V0) $(MAKE) -j TARGET=$(subst clean_,,$@) clean
-
 ## <TARGET>_clean    : clean up one specific target (alias for above)
 $(TARGETS_CLEAN):
 	$(V0) $(MAKE) -j TARGET=$(subst _clean,,$@) clean
 
 ## clean_all         : clean all valid targets
-clean_all: $(CLEAN_TARGETS)
-
-## all_clean         : clean all valid targets (alias for above)
-all_clean: $(TARGETS_CLEAN)
+clean_all: $(TARGETS_CLEAN) test_clean
 
-FLASH_TARGETS = $(addprefix flash_,$(VALID_TARGETS) )
+TARGETS_FLASH = $(addsuffix _flash,$(VALID_TARGETS) )
 
 ## flash_<TARGET>    : build and flash a target
```

Comment:
I think you need to edit this comment as well. <TARGET>_flash

Reply:
Well spotted, fixed.